spring-cloud-gcp-pubsub-stream-binder/src/test/java/com/google/cloud/spring/stream/binder/pubsub/PubSubEmulator.java

```diff
@@ -198,13 +197,13 @@ private void startEmulator() throws IOException, InterruptedException {
 
     try {
       this.emulatorProcess =
-          new ProcessBuilder("gcloud", "beta", "emulators", "pubsub", "start").start();
+          new ProcessBuilder("bash", "-c", "gcloud beta emulators pubsub start").start();
```

Comment:
Could you clarify why this change is needed?

Reply:
It's a convenience for `bash` emulators. When using the bash [installation guide](https://cloud.google.com/sdk/docs/install#installation_instructions), it alters `.bashrc` by adding the gcloud path, but it does not add `gcloud` to the global `PATH` (this is a problem in Windows). Using `bash` as the main executable instead of `gcloud` implies all that environment setup. 
Regardless, since it's just a convenience, it makes sense to discuss and implement in another PR.